### ISSUE TITLE
Cloudinary select option is undefined in the react script

### ISSUE THREAD

**Issue by timboslice69** [keystonejs/keystone#1340]
Have been troubleshooting why when i use the select option for a CloudinaryImage type the select list does not show up. As far as I've gotten the properties select, selectPreix and folder are not available in the react script in this.props so the check that select is true to show the select box always fails. Can someone enlighten me?

In my model: 

    image: { 
		type: Types.CloudinaryImage,
		folder: 'people',
		selectPrefix: 'people',
		select: true
	}

Problem Code:

	keystone/fields/cloudinaryimage/CloudinaryImageField.js
	Line 258:			{this.props.select && this.renderImageSelect()}

this.props.select is always undefined.

**Comment by azterix** [comment]
I'm still new to keystone, but I did some digging and it appears that there are a couple of things required to make the select option persist down to the template in 'this.props'.

1: The additional field properties don't seem to be defined in CloudinaryImageType.js. I added the following at line 24:

	this._properties = ['select'];

2: 'Additional' options don't seem to be picked up in the superclass defined in Type.js. I modified it as follows.

Original (starts at line 97):
```javascript
        optionKeys.forEach(function(key) {
			if (this[key]) {
				this.__options[key] = this[key];
			}
		}.bind(this));
```

My change:
```javascript
        optionKeys.forEach(function(key) {
			if (this[key]) {
				this.__options[key] = this[key];
			} else if(this.options[key]){
				this.__options[key] = this.options[key];				
			}
		}.bind(this));
```

I'm not sure if this is the right way to go about the fix as I'm still getting up to speed with the framework, so definitely test thoroughly if you use this...
